hathor/p2p/sync_v2/mempool.py

```diff
@@ -52,21 +54,29 @@ def is_running(self) -> bool:
         """Whether the sync-mempool is currently running."""
         return self._is_running
 
-    def run(self) -> None:
+    def run(self) -> Deferred[None]:
         """Starts _run in, won't start again if already running."""
```

Comment:
The doctstring should be updated with the purpose of the deferred. From what I understood it's purely so we know that mempool sync finished successfully, but no further information is given, and the deferred will never complete unless `self.missing_tips` is zeroed. Is that correct?

Reply:
Indeed. However, it's important to note that by the time we reach this stage, our node will have activated real-time relay. Consequently, even during mempool synchronization, the other peer will continue to relay new blocks and transactions, ensuring we remain updated.